.NET/Microsoft.Recognizers.Text.DateTime/Extractors/BaseMergedDateTimeExtractor.cs

```diff
@@ -270,6 +278,15 @@ private bool ShouldSkipFromToMerge(ExtractResult er)
             return config.FromToRegex.IsMatch(er.Text);
         }
 
+        /*Under TasksMode: Should not treat a four-digit number as a daterange if the input text does not include a month or year reference.
+          It should not treat 2005 as a daterange in statements like "Milk 2005."
+         (The year 2005 should be treated as a number only.)
+        */
+        private bool ShouldSkipOnlyYear(ExtractResult er)
```

Comment:
No need to do this in this PR, but it would be interesting if this mode-specific behaviours/code is isolated into it's own module/strategy.
Please move it along with new changes in a followup PR.